Tests/NimbleHelpers.swift

```diff
@@ -0,0 +1,18 @@
+import Nimble
+
+/// A Nimble matcher that succeeds when at least one of the substrings
+public func containOne(of substrings: String...) -> Predicate<String> {
+    return containOne(of: substrings)
+}
+
+/// A Nimble matcher that succeeds when at least one of the substrings
+public func containOne(of substrings: [String]) -> Predicate<String> {
```

Comment:
This could be upstreamed to nimble :))
what you think?

Reply:
I was thinking about it, but then I noticed a PR that allows concatenating expectations: https://github.com/Quick/Nimble/pull/493. This way we could introduce `or` and make it more generic. Already bumped the PR to see if we can get it updated & reviewed.